plugins/net_plugin/include/eosio/net_plugin/net_plugin.hpp

```diff
@@ -26,7 +27,7 @@ namespace eosio {
         net_plugin();
         virtual ~net_plugin();
 
-        APPBASE_PLUGIN_REQUIRES((chain_plugin))
+        APPBASE_PLUGIN_REQUIRES((chain_plugin)(producer_plugin))
```

Comment:
This should have always been here, but is now needed because of the call to `producer_plugin` in `net_plugin::plugin_initialize`.